<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPanel</class>
 <widget class="QWidget" name="AdminPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Настройки программы</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QTabWidget" name="tabWidget_2">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Параметры связи</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>Параметры связи с базой данных</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QGroupBox" name="groupBox_6">
               <property name="minimumSize">
                <size>
                 <width>500</width>
                 <height>0</height>
                </size>
               </property>
               <property name="title">
                <string>SQLite(локальная)</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QLabel" name="userLabel_3">
                    <property name="text">
                     <string>Локальный файл</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="userEdit_3">
                    <property name="text">
                     <string>c:\lely\db-lely3.db</string>
                    </property>
                    <property name="placeholderText">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>356</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_8">
               <property name="title">
                <string>PostgreSql(удаленный сервер)</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <widget class="QLabel" name="userLabel_2">
                    <property name="text">
                     <string>Логин:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="userEdit_2">
                    <property name="text">
                     <string>postgres</string>
                    </property>
                    <property name="placeholderText">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item>
                   <widget class="QLabel" name="passwordLabel_2">
                    <property name="text">
                     <string>Пароль:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="passwordEdit_2">
                    <property name="inputMask">
                     <string/>
                    </property>
                    <property name="text">
                     <string>000</string>
                    </property>
                    <property name="echoMode">
                     <enum>QLineEdit::Password</enum>
                    </property>
                    <property name="placeholderText">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_11">
                  <item>
                   <widget class="QLabel" name="databaseLabel_2">
                    <property name="text">
                     <string>База данных:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="dbnameEdit_2">
                    <property name="text">
                     <string>vim</string>
                    </property>
                    <property name="placeholderText">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_12">
                  <item>
                   <widget class="QLabel" name="hostLabel_2">
                    <property name="text">
                     <string>IP Address:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="hostEdit_2">
                    <property name="text">
                     <string>192.168.124.135</string>
                    </property>
                    <property name="placeholderText">
                     <string>localhost</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Меню</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QGroupBox" name="groupBox_7">
            <property name="title">
             <string>Добавление/удаление пунктов меню(XML)</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QTextEdit" name="textEdit">
               <property name="documentTitle">
                <string/>
               </property>
               <property name="placeholderText">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Админ панель DB</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Дерево базы данных</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QTreeView" name="treeView">
               <property name="maximumSize">
                <size>
                 <width>30000</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Выберите действие</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Таблицы данных</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QTabWidget" name="tabWidget">
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <widget class="QWidget" name="tab">
                  <attribute name="title">
                   <string>Tab 1</string>
                  </attribute>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>Отчеты Word/Excel</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QGroupBox" name="groupBox_9">
            <property name="title">
             <string>MS Word</string>
            </property>
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>26</y>
               <width>229</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Укажите путь к шаблону Word документа</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>50</y>
               <width>133</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_10">
            <property name="title">
             <string>MS Excel</string>
            </property>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>30</y>
               <width>231</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Укажите путь к шаблону Excel документа</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>60</y>
               <width>231</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>Графики</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QGroupBox" name="groupBox_11">
            <property name="title">
             <string>Настройки</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>отменить изменения</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Применить изменения</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
